src/commands/git-metadata/upload.ts

```diff
@@ -216,7 +217,7 @@ export class UploadCommand extends Command {
   private getApiRequestBuilder(apiKey: string): RequestBuilder {
     return getRequestBuilder({
       apiKey,
-      baseUrl: 'https://' + apiHost,
+      baseUrl: getBaseIntakeUrl('api'),
```

Comment:
I don't think we should call the API endpoint an intake, those are 2 different things

Reply:
```suggestion
      baseUrl: `https://api.${getDatadogSite()}`,
```